geo/CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Update proj dependency to 0.29.0
```

Comment:
```suggestion
- BREAKING: Update proj dependency to 0.29.0
```

I think we re-export a bunch of proj symbols, so technically breaking, right?

(We already have a bunch of breaking changes in this release, so no big deal)

Reply:
BTW, I like to do this as a way of accounting when doing the release if it needs to be a minor or major release. 

In this case it doesn't matter because we already have a bunch of other breaking changes, but I feel it's good hygiene, if not a bit pedantic. 🤓

Reply:
Ah, good catch!